app/controllers/api/v0/admin/users_controller.rb

```diff
@@ -0,0 +1,23 @@
+module Api
+  module V0
+    module Admin
```

Comment:
I'm won't insist on keeping the `Admin` namespace but it made sense to me to indicate that this route requires a token belonging to an admin user.